lib/resources/r4/related_person_patch.yaml

```diff
@@ -289,3 +289,16 @@ operations:
         <li>If no suffix value is provided, the suffix on the name will be unset.</li>
         <li>Only one suffix value may be provided.</li>
       </ul>
+
+  - name: test-telecom-id
```

Comment:
Can we move this to be under add-telecom so that all telecom operations are grouped together?

Reply:
Will need to update r4_examples_relatedperson.rb with the new order.

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/commit/ffb07073bcb9a4045fc1d1f226db3a3f287adeef